image/hostapd_manager.sh

```diff
@@ -61,58 +74,55 @@ confirm() {
 	esac
 }
 
+function cleanhostapd () {
+	wLog "Cleaning hostapd config at $1"
+	for j in "${HOSTAPD_SECURE_VALUES_DELETE[@]}"
+	do
+		sed -i "/$j/ d" ${1}
+	done
+	sed -i '/^\s*$/d' ${1}
+}
+
+function writehostapd () {
+	wLog "Writing hostapd config at $1"
+	sed -i '/^\s*$/d' ${1}
+	echo "" >> ${1}
+	for j in "${HOSTAPD_SECURE_VALUES_WRITE[@]}"
+	do 
+		echo "${j}" >> ${1}
+	done
+	echo "wpa_passphrase=$wifiPass" >> ${1}
+}
 
 #apply settings and restart all processes
-function APPLYSETTINGSLOUD {
+function APPLYSETTINGS {
+	wLog "Restarting all wifi settings."
 	echo "${RED}${BOLD} $att At this time the script will restart your WiFi services.${WHITE}${NORMAL}"
 	echo "If you are connected to Stratux through the ${BOLD}192.168.10.1${NORMAL} interface then you will be disconnected"
-	echo "Please wait 1 min and look for the new SSID on your wireless device."
+	echo "Please wait upto 1 min and look for the new SSID on your wireless device."
 	sleep 3
 	echo "${YELLOW}$att Restarting Stratux WiFi Services... $att ${WHITE}"
-	echo "${YELLOW}$att SSH will now  disconnect if connected to http://192.168.1.10 ... $att ${WHITE}"
```

Comment:
This is not needed... ifup wlan0 takes care of restarting all the services. see: stratux-wifi.sh